analysis_options.yaml

```diff
@@ -0,0 +1,101 @@
+analyzer:
+  strong-mode:
+    implicit-dynamic: False
+    implicit-casts: False
+  exclude:
+    - test
+
+linter:
```

Comment:
Where did this list of linters come from? Why does vector_math want any of them? How do we decide when to add / remove from this list? How regularly do we update this list?